assets/components/trackedComponent/trackedComponent.jsx

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+// ----- Imports ----- //
+
+import * as React from 'react';
+import { trackComponentEvents } from 'helpers/tracking/ophanComponentEventTracking';
+import type { OphanComponent, OphanAction } from 'helpers/tracking/ophanComponentEventTypes';
+
+// ----- Prop Types ----- //
+
+type Props = {
+  component: OphanComponent,
+  action: OphanAction,
+  children?: React.Node,
+};
+
+
+// ----- Component ----- //
+
+class TrackedComponent extends React.Component<Props> {
```

Comment:
Since it is the first non-functional component, Is it worth to explain in a comment why we need this? Additionally, it would be a good idea to mention in the comment to avoid this kind of components when possible.